docs/source/bond_family_guide/design/object_model/organization.rst

```diff
@@ -0,0 +1,114 @@
+..
+   INTEL CONFIDENTIAL
+   Copyright(c) 2016 Intel Corporation All Rights Reserved.
+
+   The source code contained or described herein and all documents related to
+   the source code ("Material") are owned by Intel Corporation or its suppliers
+   or licensors. Title to the Material remains with Intel Corporation or its
+   suppliers and licensors. The Material contains trade secrets and proprietary
+   and confidential information of Intel or its suppliers and licensors. The
+   Material is protected by worldwide copyright and trade secret laws and
+   treaty provisions. No part of the Material may be used, copied, reproduced,
+   modified, published, uploaded, posted, transmitted, distributed, or
+   disclosed in any way without Intel's prior express written permission.
+
+   No license under any patent, copyright, trade secret or other intellectual
+   property right is granted to or conferred upon you by disclosure or delivery
+   of the Materials, either expressly, by implication, inducement, estoppel or
+   otherwise. Any license under such intellectual property rights must be
+   express and approved by Intel(R) in writing.
+
+.. _object_type_organization:
+
+object-type: organization
+=========================
+
+Attributes
+----------
+
+  object-id
+    See :ref:`common_attributes`.
+
+  object-type
+    The string 'organization'.
+ 
+  creator-id
+    See :ref:`common_attributes`.
+
+  ref-count
+      See :ref:`common_attributes`.
+
+  name
+    The name of the organization.
+
+    Unique, Indexed
+
+  industry
+    The industry of the organization.
+
+    Optional
+
+  ticker
+    The stock market ticker symbol associated with the organization.
```

Comment:
From testing, we seem to put no constraints on the ticker symbols. 
Deferring this, and not changing. 
A quick search of the code was not able to answer this question.